charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -70,7 +70,7 @@ resources:
 
 priorityClassName: ""
 nodeSelector: {}
-tolerateAllTaints: true
+tolerateAllTaints: false
```

Comment:
Why do we have these changes? Release commit should only have the version update.

Reply:
yeah, in retrospect, this should have been a separate commit. will split it out into two commits next time.